### PR TITLE
feat(integrations): Perplexity support

### PR DESCRIPTION
**Pull request by Shubhamai**
## Describe your changes
Adding [perpexity](http://perplexity.ai) support

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: docs-v2/integrations/all/perplexity.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | ✅ |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [Perplexity API docs](https://docs.perplexity.ai/api-reference/chat-completions/)
+
```
Comment (hassan254-prog) [inline docs-v2/integrations/all/perplexity.mdx:25]: - Would be nice to add  [Perplexity rate limiting](https://docs.perplexity.ai/guides/rate-limits)

- Can you also add a link to where you generated the `APIkey`, i.e [Generate an apiKey in your Perplexity account]

File: packages/shared/providers.yaml [hunk 2]
```diff
@@ -3460,6 +3460,23 @@ peopledatalabs:
...
+        headers:
+            Authorization: Bearer ${apiKey}
+        base_url: https://api.perplexity.ai
+    docs: https://docs.nango.dev/integrations/all/perplexity
+    credentials:
+        apiKey:
+            type: string
+            title: API Key
+            description: The API key for your Perplexity account
+
```
Comment (hassan254-prog) [inline packages/shared/providers.yaml:3564]: Please introduce the following fields:
```
pattern: '^pplx-[a-f0-9]+$'
example: pplx-xxxxxx

**Review by hassan254-prog — COMMENTED** [review]
Thanks @Shubhamai  for your contribution, small comments but looks good overall.
I prepared a [logo](https://codefile.io/f/RY96UE7PjU) for you

**Comment by Shubhamai** [comment]
Thanks a lot for the review. Just made the following changes.

File: docs-v2/integrations/all/perplexity.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (API Key)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | ✅ |
```
Comment (khaliqgant) [inline docs-v2/integrations/all/perplexity.mdx]:
```suggestion
| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
```

**Review by khaliqgant — APPROVED** [review]

**Review by hassan254-prog — APPROVED** [review]

**Comment by Shubhamai** [comment]
Hi, thanks for the merge. I am still unable to see perplexity provider in the [Integrations page](https://app.nango.dev/dev/integrations/create) while it is in the [docs](https://docs.nango.dev/integrations/all/perplexity). [Elevenlabs provider](https://github.com/NangoHQ/nango/pull/2798) was shown much quicker in the integrations page.

**Comment by khaliqgant** [comment]
> Hi, thanks for the merge. I am still unable to see perplexity provider in the [Integrations page](https://app.nango.dev/dev/integrations/create) while it is in the [docs](https://docs.nango.dev/integrations/all/perplexity). [Elevenlabs provider](https://github.com/NangoHQ/nango/pull/2798) was shown much quicker in the integrations page.

No problem !  We have yet to do a deploy. Will ping here once it has been deployed

**Comment by khaliqgant** [comment]
@Shubhamai this has been deployed.

**Pull request merged**